docs/api/serviceregistry/service-registry-management-generic-http.md

```diff
@@ -0,0 +1,1524 @@
+}
+```
+
+### interface-template-create
+
+The service operation **request** requires an authorization bearer header and an [InterfaceTemplateListRequest](../data-models/interface-template-list-request.md) JSON encoded body.
+
+```
+POST /serviceregistry/mgmt/interface-templates HTTP/1.1
+Authorization: Bearer <authorization-info>
+
+{
+  "interfaceTemplates": [
+    {
+      "name": "ftp",
+      "protocol": "tcp",
+      "propertyRequirements": [
+        {
+          "name": "accessAddresses",
+          "mandatory": true,
+          "validator": "not_empty_address_list",
+          "validatorParams": [
+          ]
+        }
+      ]
+    },
+    {
+      "name": "sftp",
+      "protocol": "tcp",
+      "propertyRequirements": [
+        {
+          "name": "accessAddresses",
+          "mandatory": true,
+          "validator": "not_empty_address_list",
+          "validatorParams": [
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+The service operation **responds** with the status code `201` if the interface template entities were successfully created. The response also contains an [InterfaceTemplateListResponse](../data-models/interface-template-list-response.md) JSON encoded body.
+
+```
+{
+  "entries": [
+    {
+      "name": "sftp",
+      "protocol": "tcp",
+      "propertyRequirements": [
+        {
+          "name": "accessAddresses",
+          "mandatory": true,
+          "validator": "NOT_EMPTY_ADDRESS_LIST",
+          "validatorParams": []
+        }
+      ],
+      "createdAt": "2025-03-15T23:09:00.882593100Z",
+      "updatedAt": "2025-03-15T23:09:00.882593100Z"
+    },
+    {
+      "name": "ftp",
+      "protocol": "tcp",
+      "propertyRequirements": [
+        {
+          "name": "accessAddresses",
+          "mandatory": true,
+          "validator": "NOT_EMPTY_ADDRESS_LIST",
+          "validatorParams": []
+        }
+      ],
+      "createdAt": "2025-03-15T23:09:00.892877600Z",
+      "updatedAt": "2025-03-15T23:09:00.892877600Z"
+    }
+  ],
+  "count": 2
+}
+```
+
+The **error codes** are, `400` if the request is malformed, `401` if the requester authentication was unsuccessful, `403` if the authenticated requester has no permission and `500` if an unexpected error happens. The error response also contains an [ErrorResponse](../data-models/error-response.md) JSON encoded body.
+
+```
+{
+  "errorMessage": "Interface template already exists: ftp",
+  "errorCode": 400,
+  "exceptionType": "INVALID_PARAMETER",
+  "origin": "POST /serviceregistry/mgmt/interface-templates"
+}
+```
+
+### interface-template-remove
+
+The service operation **request** requires an authorization bearer header and a List<[InterfaceTemplate](../primitives.md#interfacetemplate)> as path parameter, which contains the string identifier of the interface descriptors that need to be removed.
+
+```
+DELETE /serviceregistry/mgmt/interface-templates?names=ftp&names=sftp HTTP/1.1
```

Comment:
`ftp` and `sftp` are protocols and not template names. A right example would be `custom-ftp` or `my-awesome-ftp`.